docs/source/conf.py

```diff
@@ -63,9 +63,9 @@
 master_doc = 'contents'
 
 # General information about the project.
-project = u'Sawtooth'
-copyright = u'2015-2017, Intel Corporation'
-author = u'Intel Corporation'
+project = u'Sawtooth Seth'
+copyright = u'2017-2018'
+author = u'Hyperledger Foundation'
```

Comment:
Awaiting clarification from Linux Foundation / Hyperledger on the copyright policy.

Reply:
I'll revert these for now, so we can get this merged sooner @dcmiddle

Reply:
sounds good

Reply:
@dcmiddle I have reverted back to using the original copyright notices